Model/util: update `SampleDataUtil`

Creastery opened the pull request:
Let's update the `SampleDataUtil` to populate our application with more
sample data to facilitate testing, and reduce time spent by users to
create their customised degree plan using our application.

Creastery commented:
Seems like a bug with Travis. AppVeyor and local tests (both GUI and headless) are passing.

WendyBaiYunwei commented on src/main/java/seedu/address/model/util/SampleDataUtil.java:
```diff
@@ -60,16 +69,202 @@
+    );
+
+    private static final Module CS3235 = new Module(
+            new Name("Introduction to Computer Security"),
```
I am not sure whether following the real name is necessary, but CS3235 should be "Computer Security" according to NUSMods.

Creastery replied:
We are following details from NUS website, which clearly states "CS3235 Introduction to Computer Security".

WendyBaiYunwei commented on src/main/java/seedu/address/model/util/SampleDataUtil.java:
If the credits here is referring to the actual total credits requirement of a Information Security Requirements category, then the credits should be "32", according to official web page for prospective students.

If the credits refers to total credits for the following input modules, then credits should be 16.

WendyBaiYunwei replied:
For the following comments, I will assume that the credits refers to the former. (actual total credits)

Creastery replied:
> If the credits here is referring to the actual total credits requirement of a Information Security Requirements category, then the credits should be "32", according to official web page for prospective students.

We already split `Information Security Requirements` to `Information Security Requirements` (compulsory modules to be taken) and `Information Security Electives` (elective modules to be taken) since PR #77 (prior to milestone v1.2).

> If the credits refers to total credits for the following input modules, then credits should be 16.

We are not keeping using this field to keep track of the current credits in the module. We are keeping track of the total credits required to fulfill the requirement category.

WendyBaiYunwei commented on src/main/java/seedu/address/model/util/SampleDataUtil.java:
Perhaps students from poly has different requirements as students from JC do. But according to https://www.comp.nus.edu.sg/programmes/ug/isc/curr/, credits for UE should be 32. While we may not need to follow the real data, it is perhaps better to standardize as most of the sample data are real.

Creastery replied:
You are right. I'll include a fix for the erroneous data from PR #77 while I'm at it.

WendyBaiYunwei reviewed (changes requested):
I am not sure whether it is intended to populate real data. In the review, I assumed that real data should be populated.

WendyBaiYunwei reviewed (commented):
Please tell me when I can merge for you the PR. The current PR seems ok.

Creastery commented:
Delaying this PR to v1.4. Need further tests to determine why only Travis is failing to build properly.

Creastery committed:
Model/util: update `SampleDataUtil`

Let's update the `SampleDataUtil` to populate our application with more
sample data to facilitate testing, and reduce time spent by users to
create their customised degree plan using our application.

Creastery commented:
Thanks for reviewing!
